lib/_stream_readable.js

```diff
@@ -1139,3 +1139,31 @@ function endReadableNT(state, stream) {
     }
   }
 }
+
+Readable.from = function(iterator, opts) {
+  const readable = new Readable(opts);
+  readable._read = function() {
+    next().catch(onError);
+  };
+  function onError(err) {
+    readable.destroy(err);
+  }
+  let curNext;
```

Comment:
Add note on why this is required? (since it was an issue in the utils library discussion)

Reply:
Sure I've added a comment on this.